2020-11-18 22:47:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL """UPDATE USUARIOS SET DOCUMENTO=1144096206, NOMBRE=USUARIO6, APELLIDO=PREUBA6, CORREO=PRUEBA6@[*]MAIL.COM, DIRECCION= XXXX CR CCC, TELEFONO=3137739635, PASSWORD=TEST6 WHERE CODE=6 ""[*]"; se esperaba "UPDATE, USE, {"
Syntax error in SQL statement """UPDATE USUARIOS SET DOCUMENTO=1144096206, NOMBRE=USUARIO6, APELLIDO=PREUBA6, CORREO=PRUEBA6@[*]MAIL.COM, DIRECCION= XXXX CR CCC, TELEFONO=3137739635, PASSWORD=TEST6 WHERE CODE=6 ""[*]"; expected "UPDATE, USE, {"; SQL statement:
"UPDATE USUARIOS SET DOCUMENTO=1144096206, NOMBRE=USUARIO6, APELLIDO=PREUBA6, CORREO=PRUEBA6@[*]MAIL.COM, DIRECCION= XXXX CR CCC, TELEFONO=3137739635, PASSWORD=TEST6 WHERE CODE=6 " [42001-197]
2020-11-18 22:49:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL """update usuarios ser documento=1144096206, nombre=USUARIO6, apellido=PREUBA6, correo=PRUEBA6@MAIL.COM, direccion= XXXX CR CCC, telefono=3137739635, password=TEST6 WHERE code=6 ""[*]"; se esperaba "UPDATE, USE, {"
Syntax error in SQL statement """update usuarios ser documento=1144096206, nombre=USUARIO6, apellido=PREUBA6, correo=PRUEBA6@MAIL.COM, direccion= XXXX CR CCC, telefono=3137739635, password=TEST6 WHERE code=6 ""[*]"; expected "UPDATE, USE, {"; SQL statement:
"update usuarios ser documento=1144096206, nombre=USUARIO6, apellido=PREUBA6, correo=PRUEBA6@MAIL.COM, direccion= XXXX CR CCC, telefono=3137739635, password=TEST6 WHERE code=6 " [42001-197]
2020-11-18 22:50:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL """update usuarios set documento=1144096206, nombre=USUARIO6, apellido=PREUBA6, correo=PRUEBA6@MAIL.COM, direccion= XXXX CR CCC, telefono=3137739635, password=TEST6 WHERE code=6 ""[*]"; se esperaba "UPDATE, USE, {"
Syntax error in SQL statement """update usuarios set documento=1144096206, nombre=USUARIO6, apellido=PREUBA6, correo=PRUEBA6@MAIL.COM, direccion= XXXX CR CCC, telefono=3137739635, password=TEST6 WHERE code=6 ""[*]"; expected "UPDATE, USE, {"; SQL statement:
"update usuarios set documento=1144096206, nombre=USUARIO6, apellido=PREUBA6, correo=PRUEBA6@MAIL.COM, direccion= XXXX CR CCC, telefono=3137739635, password=TEST6 WHERE code=6 " [42001-197]
2020-11-18 22:51:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "UPDATE USUARIOS SET DOCUMENTO=1144096206, NOMBRE=USUARIO6, APELLIDO=PREUBA6, CORREO=PRUEBA6@[*]MAIL.COM, DIRECCION= XXXX CR CCC, TELEFONO=3137739635, PASSWORD=TEST6 WHERE CODE=6 "
Syntax error in SQL statement "UPDATE USUARIOS SET DOCUMENTO=1144096206, NOMBRE=USUARIO6, APELLIDO=PREUBA6, CORREO=PRUEBA6@[*]MAIL.COM, DIRECCION= XXXX CR CCC, TELEFONO=3137739635, PASSWORD=TEST6 WHERE CODE=6 "; SQL statement:
update usuarios set documento=1144096206, nombre=USUARIO6, apellido=PREUBA6, correo=PRUEBA6@MAIL.COM, direccion= XXXX CR CCC, telefono=3137739635, password=TEST6 WHERE code=6 [42000-197]
2020-11-18 22:52:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "UPDATE USUARIOS
SET DOCUMENTO=1144096206,
      NOMBRE=USUARIO6, 
      APELLIDO=PREUBA6,
      CORREO=PRUEBA6@[*]MAIL.COM,
      DIRECCION=XXXX CR CCC,
      TELEFONO=3137739635,
     PASSWORD=TEST6
     WHERE CODE=6 "
Syntax error in SQL statement "UPDATE USUARIOS
SET DOCUMENTO=1144096206,
      NOMBRE=USUARIO6, 
      APELLIDO=PREUBA6,
      CORREO=PRUEBA6@[*]MAIL.COM,
      DIRECCION=XXXX CR CCC,
      TELEFONO=3137739635,
     PASSWORD=TEST6
     WHERE CODE=6 "; SQL statement:
update usuarios
set documento=1144096206,
      nombre=USUARIO6, 
      apellido=PREUBA6,
      correo=PRUEBA6@MAIL.COM,
      direccion=XXXX CR CCC,
      telefono=3137739635,
     password=TEST6
     where code=6 [42000-197]
2020-11-18 22:52:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "UPDATE USUARIOS
SET DOCUMENTO=1144096206,
      NOMBRE=USUARIO6, 
      APELLIDO=PREUBA6,
      CORREO=PRUEBA6@[*]MAIL.COM,
      DIRECCION=XXXX CR CCC,
      TELEFONO=3137739635,
     PASSWORD=TEST6
     WHERE CODE=6 "
Syntax error in SQL statement "UPDATE USUARIOS
SET DOCUMENTO=1144096206,
      NOMBRE=USUARIO6, 
      APELLIDO=PREUBA6,
      CORREO=PRUEBA6@[*]MAIL.COM,
      DIRECCION=XXXX CR CCC,
      TELEFONO=3137739635,
     PASSWORD=TEST6
     WHERE CODE=6 "; SQL statement:
update usuarios
set documento=1144096206,
      nombre=USUARIO6, 
      apellido=PREUBA6,
      correo=PRUEBA6@MAIL.COM,
      direccion=XXXX CR CCC,
      telefono=3137739635,
     password=TEST6
     where code=6 [42000-197]
2020-11-18 22:52:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "UPDATE USUARIOS
SET DOCUMENTO=1144096206,
      NOMBRE=USUARIO6, 
      APELLIDO=PREUBA6,
      CORREO=PRUEBA6@[*]MAIL.COM,
      DIRECCION=XXXX CR CCC,
      TELEFONO=3137739635,
     PASSWORD=TEST6
     WHERE CODE=6 "
Syntax error in SQL statement "UPDATE USUARIOS
SET DOCUMENTO=1144096206,
      NOMBRE=USUARIO6, 
      APELLIDO=PREUBA6,
      CORREO=PRUEBA6@[*]MAIL.COM,
      DIRECCION=XXXX CR CCC,
      TELEFONO=3137739635,
     PASSWORD=TEST6
     WHERE CODE=6 "; SQL statement:
update usuarios
set documento=1144096206,
      nombre=USUARIO6, 
      apellido=PREUBA6,
      correo=PRUEBA6@MAIL.COM,
      direccion=XXXX CR CCC,
      telefono=3137739635,
     password=TEST6
     where code=6 [42000-197]
2020-11-18 22:53:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "USUARIOS[*]"; se esperaba "UPDATE, USE, {"
Syntax error in SQL statement "USUARIOS[*]"; expected "UPDATE, USE, {"; SQL statement:
USUARIOS [42001-197]
2020-11-19 08:34:42 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Columna "CORRE" no encontrada
Column "CORRE" not found; SQL statement:
SELECT * FROM USUARIOS where corre='prueba1@mail.com' and password='test1' [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.expression.Comparison.optimize(Comparison.java:177)
	at org.h2.expression.ConditionAndOr.optimize(ConditionAndOr.java:130)
	at org.h2.command.dml.Select.prepare(Select.java:861)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-11-19 08:50:08 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Error de Sintaxis en sentencia SQL "SELECT * FROM USUARIOS 
INSERT INTO[*] USUARIOS VALUES ('1144096301','usuario100','preuba100','prueba100@mail.com',' XXXX Cr CCC','3137739635','test100')"
Syntax error in SQL statement "SELECT * FROM USUARIOS 
INSERT INTO[*] USUARIOS VALUES ('1144096301','usuario100','preuba100','prueba100@mail.com',' XXXX Cr CCC','3137739635','test100')"; SQL statement:
SELECT * FROM USUARIOS 
INSERT INTO USUARIOS VALUES ('1144096301','usuario100','preuba100','prueba100@mail.com',' XXXX Cr CCC','3137739635','test100') [42000-197]
2020-11-19 08:50:26 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: La cantidad de columnas no coincide
Column count does not match; SQL statement:
INSERT INTO USUARIOS VALUES ('1144096301','usuario100','preuba100','prueba100@mail.com',' XXXX Cr CCC','3137739635','test100') [21002-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:306)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-11-19 09:06:48 jdbc[6]: null
org.h2.message.DbException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-197]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:197)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:733)
	at org.h2.server.web.WebApp.login(WebApp.java:954)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:134)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.h2.jdbc.JdbcSQLException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 14 more
2020-11-19 09:06:48 jdbc[6]: null
org.h2.message.DbException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-197]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:197)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:733)
	at org.h2.server.web.WebApp.login(WebApp.java:954)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:134)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.h2.jdbc.JdbcSQLException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 14 more
